tests/models/colqwen2/test_processing_colqwen2.py

```diff
@@ -0,0 +1,250 @@
+import shutil
+import tempfile
+import unittest
+
+import torch
+
+from transformers import AutoProcessor, Qwen2VLProcessor
+from transformers.models.colqwen2.processing_colqwen2 import ColQwen2Processor
+from transformers.testing_utils import get_tests_dir, require_torch, require_vision
+from transformers.utils import is_vision_available
+
+from ...test_processing_common import ProcessorTesterMixin
+
+
+if is_vision_available():
+    from transformers import (
+        ColQwen2Processor,
+    )
+
+SAMPLE_VOCAB = get_tests_dir("fixtures/test_sentencepiece.model")
+
+
+@require_vision
```

Comment:
let's have 
```suggestion
@require_torch
@require_vision
```
here and remove all other @require on the methods